pom.xml

```diff
@@ -28,8 +28,8 @@
         <version.arquillian>1.1.11.Final</version.arquillian>
         <version.org.wildfly.arquillian>1.0.2.Final</version.org.wildfly.arquillian>
         <version.arquillian.governor>1.0.1.Final</version.arquillian.governor>
-        <version.eap6>7.5.11.Final-redhat-SNAPSHOT</version.eap6>
-        <version.eap7>7.0.0.GA-redhat-2</version.eap7> <!-- eap-7.0.0 -->
+        <version.eap6>7.5.11.Final-redhat-SNAPSHOT</version.eap6> <!--eap 6.4.x downstream -->
+        <version.eap7>7.0.3.GA-redhat-SNAPSHOT</version.eap7> <!-- eap-7 downstream-->
         <org.jboss.bom.version>7.0.0.GA</org.jboss.bom.version>
```

Comment:
I don't think it is a good idea until the changes needed to the integration tests get working is not merged, For now I think we can just do like Kevin suggested, fork in within our repo and use it.